hail/src/main/scala/is/hail/backend/local/LocalBackend.scala

```diff
@@ -28,6 +28,14 @@ import scala.reflect.ClassTag
 
 class LocalBroadcastValue[T](val value: T) extends BroadcastValue[T] with Serializable
 
+class LocalTaskContext(val partitionId: Int) extends HailTaskContext {
+  override type BackendType = LocalBackend
+
+  override def stageId(): Int = ???
+
+  override def attemptNumber(): Int = ???
```

Comment:
Not sure what the implementation of these is outside of Spark

Reply:
they can be 0 and 0, due to our design of the local backend